molgenis-navigator/src/main/frontend/src/store/actions.js

```diff
@@ -59,34 +59,21 @@ function toEntity (item: any) {
 }
 
 /**
- * Get all ( first 1000 ) packages from the server
```

Comment:
the fact we only get the first 1000 items seems important to me, i suggest to keep this as part of the js doc, as this is not derivable from the function name or params